src/main/java/de/bonndan/nivio/assessment/kpi/CustomKPI.java

```diff
@@ -0,0 +1,138 @@
+package de.bonndan.nivio.assessment.kpi;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.bonndan.nivio.ProcessingException;
+import de.bonndan.nivio.assessment.Status;
+import de.bonndan.nivio.assessment.StatusValue;
+import de.bonndan.nivio.model.Labeled;
+import org.apache.commons.lang3.Range;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * A configured key performance indicator related to a landscape item label.
+ */
+public class CustomKPI extends KPI {
```

Comment:
Renaming ``KPI`` to ``KeyPerformanceIndicator`` would make 100% clear what is meant.

Reply:
But we need to save precious bytes!